relay-server/src/services/processor.rs

```diff
@@ -1041,9 +1083,9 @@ impl EnvelopeProcessorService {
         Ok(())
     }
 
-    fn light_normalize_event(
+    fn light_normalize_event<G>(
```

Comment:
nit: We could already introduce marker traits here (without any methods), e.g. `EventGroup` implemented by  `TransactionGroup` and `ErrorGroup`.

```suggestion
    fn light_normalize_event<G: EventGroup>(
```

Reply:
yap, that what I was mentioning today in the morning.  I can also do in in the followups, to have fewer changes in this PR.

Reply:
I'd prefer follow up 👍